client/src/api/atlas.ts

```diff
@@ -20,8 +20,12 @@ export async function getNumberOfStudentsByYear(params: string) {
     return fetch(url).then((response) => (response.json()))
 }
 
-export async function getFiltersValues() {
-    const url = `${VITE_APP_SERVER_URL}/atlas/get-filters-values`
+export async function getFiltersValues(geoId: string) {
+    let url = `${VITE_APP_SERVER_URL}/atlas/get-filters-values`;
+    if (geoId) {
+        url = `${VITE_APP_SERVER_URL}/atlas/get-filters-values?geo_id=${geoId}`;
+    }
+
```

Comment:
const url = `${VITE_APP_SERVER_URL}/atlas/get-filters-values{geoId ? "?geo_id=${geoId}" : "}` (or something like that)